src/services/positionService.ts

```diff
@@ -248,14 +272,21 @@ export default class PositionService {
                   (position.pair.swaps[0] && parseInt(position.pair.swaps[0].executedAtTimestamp, 10)) ||
                   position.createdAtTimestamp,
                 pairNextSwapAvailableAt: position.createdAtTimestamp.toString(),
+                ...(!!position.permissions && { permissions: position.permissions }),
```

Comment:
It's not necessary to use a ternary operator since boolean values spread to nothing, example: 

<img width="368" alt="image" src="https://user-images.githubusercontent.com/113060391/228615571-92d324d7-3ef6-43c5-a855-a428336165cb.png">

However, if you prefer using a ternary operator because you believe it's clearer, I will include your suggestion and procede that way from now on forward whenever I implement a short circuit of object spreading.

Reply:
nah, lets keep it as you made it